internal/page/fixtures/donor.go

```diff
@@ -303,6 +304,19 @@ func updateLPAProgress(
 		donorDetails.Tasks.PeopleToNotify = actor.TaskCompleted
 	}
 
+	if data.Progress == slices.Index(progressValues, "addCorrespondent") {
```

Comment:
do `>=` for these

Reply:
My thinking here was to make this an explicit choice given that in the future all comms will be routed to the correspondent rather than the donor which may get confusing when setting up fixtures.

Reply:
yeah, if we are doing special cases it would be better under a different option and keep `progress` as a pretty straightforward "fills in up to this step"